src/manifolds/FixedRankMatrices.jl

```diff
@@ -1,39 +1,40 @@
-@doc doc"""
+@doc raw"""
     FixedRankMatrices{m,n,k,T} <: Manifold
 
-The manifold of $m\times n$ real-valued (complex-valued) matrices of fixed rank $k$, i.e.
+The manifold of $m \times n$ real-valued or complex-valued matrices of fixed rank $k$, i.e.
```

Comment:
Can `×` go here and below?

Reply:
Ah yes; I also checked for all other occurrences of `\times` and replaced them with `×`.